stacktrace.go

```diff
@@ -165,6 +165,10 @@ func fileContext(filename string, line, context int) ([][]byte, int) {
 	if !ok {
 		data, err := ioutil.ReadFile(filename)
 		if err != nil {
+			// cache errors as nil slice: code below handles it correctly
```

Comment:
I haven't 100% verified this. But I'd feel more comfortable adding an extra check explicitly for a `nil` value. If anything, to return early. This will prevent forgetting in the future that the value may be `nil`.

This would just be a matter of:

```go
if lines == nil {
  return nil, 0
}
```